records/state/state.go

```diff
@@ -46,9 +46,22 @@ type Label struct {
 
 // Status holds a task status as defined in the /state.json Mesos HTTP endpoint.
 type Status struct {
-	Timestamp float64 `json:"timestamp"`
-	State     string  `json:"state"`
-	Labels    []Label `json:"labels,omitempty"`
+	Timestamp       float64         `json:"timestamp"`
+	State           string          `json:"state"`
+	Labels          []Label         `json:"labels,omitempty"`
+	ContainerStatus ContainerStatus `json:"container_status"`
```

Comment:
I personally don't like to use `omitempty` because when a field is absent in a JSON payload, one can't tell the difference between the field not existing and being empty for a particular instance.